app/src/organisms/ModuleCard/TestShakeSlideout.tsx

```diff
@@ -264,51 +267,19 @@ export const TestShakeSlideout = (
           ) : null}
         </Flex>
       </Flex>
-      <Flex
-        border={`${SPACING.spacingXXS} solid ${COLORS.medGrey}`}
-        borderRadius={SPACING.spacing2}
-        flexDirection={DIRECTION_COLUMN}
-        fontWeight={TYPOGRAPHY.fontWeightRegular}
-        paddingY={SPACING.spacing4}
-        width="100%"
+      {showWizard && (
+        <HeaterShakerWizard onCloseClick={() => setShowWizard(false)} />
+      )}
+      <Link
+        marginTop={SPACING.spacing2}
+        fontSize={TYPOGRAPHY.fontSizeP}
+        fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+        color={COLORS.blue}
```

Comment:
We might have to wait until `release_6.0.0` merges into `edge` before implementing this change. But We added something in the release branch that makes the all the `links` tab-able by adding a `role="button"` and then adding a `tabIndex` to `Link`

```suggestion
      <Link
        role="button"
        marginTop={SPACING.spacing2}
         css={TYPOGRAPHY.pRegular}
        color={COLORS.blue}
```

Reply:
oof, good call @jerader. i hope that rebase isn't going to be too messy for this branch